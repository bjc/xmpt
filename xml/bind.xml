<iq type='set' id='bind'>
  <bind xmlns='urn:ietf:params:xml:ns:xmpp-bind'>
    <resource>{resourcepart}</resource>
  </bind>
</iq>
