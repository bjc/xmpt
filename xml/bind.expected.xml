<iq from='{bare_jid}' to='{full_jid}' id='bind' type='result'>
  <bind xmlns='urn:ietf:params:xml:ns:xmpp-bind'>
    <jid>{full_jid}</jid>
  </bind>
</iq>
